distribution/docs/src/main/resources/content/_managing/_configuring/configuring-idp.adoc

```diff
@@ -68,26 +68,21 @@ Run the command `feature:install security-idp` from the ${command-console}.
 </md:EntityDescriptor>
 ----
 
-.Configuring IdP as the Authentication Type
+.Configuring Authentication Types to use IdP
```

Comment:
We need to swap out all these references to IdP over to SAML unless they are talking about IdPs in general and also apply to OIDC setups.

Reply:
Just went through and verified/changed the IdP references.

As for the `also apply to OIDC setups` part, do you mean to add an OIDC section wherever there is an IdP section?

Reply:
If it makes sense to put an OIDC section in, then yes. If it is a section that generically applies to any and all IdPs whether they are SAML or OIDC, then we should mention both or neither imo.